docs/developer-guide/syntax-reference.rst

```diff
@@ -1561,7 +1565,11 @@ Scalar functions
 |                        |                                                                           | considered to be equal to any value. FIELD is the |
 |                        |                                                                           | complement to ELT.                                |
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
-| FLOOR                  |  ``FLOOR(col1)``                                                          | The floor of a value.                             |
+| FLOOR                  |  ``FLOOR(col1)`` or   ``FLOOR(col1, precision)``                          | The floor of a value.                             |
+|                        |                                                                           |                                                   |
+|                        |                                                                           | precision is a number indicating the number of    |
```

Comment:
How about: "precision is an INT indicating the number of decimal places to round down to."

Reply:
Same precision name opinion as `CEIL` (see above)